packages/pwa/app/translations/README.md

```diff
@@ -89,6 +89,21 @@ For example, in your React component, you can add formatted messages like `intl.
 
 At the minimum, only defaultMessage is the required parameter. The message id is optional. If you don’t specify it, the id is auto-generated for you.
 
+### Creating message id
```

Comment:
@vmarta I've added some instructions about message ids here. I'd like to have your thoughts on it